lib/turn.rb

```diff
@@ -0,0 +1,27 @@
+class Turn
+  attr_reader :symbol
+
+  def initialize(symbol)
+    @symbol = symbol
+  end
+
+  def get_computer_input
+    ("A".."G").to_a.sample
+  end
+
+  def get_human_input
+    gets.chomp.upcase
+  end
+
+  def get_input(symbol)
```

Comment:
This logic will have to change when we want to add the feature to include two human players. Looks great for now.